v3/src/components/case-table/case-table.tsx

```diff
@@ -1,7 +1,7 @@
 import { useDndContext } from "@dnd-kit/core"
 import { observer } from "mobx-react-lite"
 import React, { CSSProperties, useEffect, useRef } from "react"
-import { AttributeDragOverlay } from "./attribute-drag-overlay"
+import { AttributeDragOverlay } from "../utility-components/attribute-drag-overlay"
```

Comment:
How about `src/components/drag-drop/attribute-drag-overlay` instead? We may want to create a separate `tiles` folder for `graph`, `case-table`, etc. because `components` currently contains both tiles and general-use components, but `utility-components` doesn't seem quite right.